src/Components/Link.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { useSelector, useDispatch, connect } from 'react-redux';
+import {selectLoggedIn} from '../actions'; 
+
+
+const Link = ({ className, loginType, href, children }) => {
+  const dispatch = useDispatch();
+  const loggedState = useSelector(state => state.currentLoggedInAs) 
+  //console.log(`loggedState ${loggedState}`)
+
+  const onClick = (event) => { 
```

Comment:
What does this event do? doLogin? try to find a different name than "onClick"
btw, "on" is starting an event in js - don't use save or common words